Database/Corrections/sodQuestFixes.lua

```diff
@@ -135,6 +135,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78092},
             [questKeys.zoneOrSort] = sortKeys.PALADIN,
         },
+        [78114] = {
```

Comment:
`requiredSpell` is missing. We need that field to properly hide the rune after it was acquired.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5328/commits/26f950962582fe19c980069f077a4609e04859ab